app/views/static_pages/start_page.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        <%= t('student_loans.journey_name') %>
+      </h1>
+
+      <p class="govuk-body">
+        Claim student loan repayments if you:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          teach biology, chemistry, physics, computer science or languages (not including English)
+          for over half your working hours, including your planning, preparation and assessment (PPA) time
+        </li>
+        <li>have been teaching for 1 to 5 years</li>
+        <li>teach in certain places - <a href="https://www.gov.uk/guidance/teachers-student-loan-reimbursement-guidance-for-teachers-and-schools#eligibility-criteria">check which places are eligible</a>.</li>
```

Comment:
I think we can drop the full stop at the end of the li so its consistent with the others.